src/analysis/clustering.jl

```diff
@@ -167,13 +167,13 @@ function target_clusters(
     clusters::Vector{T},
     outcomes::AbstractMatrix{F};
     metric=temporal_variability,
-    size_limit=0.01
+    size_limit=0.01,

```

Comment:
Is it the auto-formatter that is adding these?

Reply:
@ConnectedSystems 
Yes, that's the auto-formatter. But I like the commas. Makes it easier to move things around. DO you want me to try to configure the auto-formatter to not add these?